rollingpin/deploy.py

```diff
@@ -82,7 +82,7 @@ def process_host(self, host, commands, timeout=0):
                 yield self.event_bus.trigger(
                     "host.command", host=host, command=command)
                 result = yield connection.execute(log, command, timeout)
-                results.append(result)
+                results.append((command, result))
```

Comment:
I find it really hard to keep track when passing around tuples like this. Is there an alternate structure which would be a little more self-documenting when away from this point in the code? Perhaps just namedtuples instead? (then we could avoid some of the unpacking later on and instead just directly reference named attributes.